docs/jwst/associations/overview.rst

```diff
@@ -126,13 +126,8 @@ as their respective DataModels.
   from jwst.datamodels import open as dm_open
   container_model = dm_open('jw12345-o001_20210311t170002_image3_001_asn.json')
 
-.. _asn-utilities:
-
 Utilities
 =========
 
-Other useful utilities for creating and manipulating associations:
-
-- :ref:`asn-from-list`
-- :ref:`asn-generate`
-- *many other TBD*
+There are a number of utilities to create user-specific associations which are
```

Comment:
Either insert a comma after "assocations", or change "which" to "that"

Reply:
✅